src/applications/gi/components/profile/EstimatedBenefits.jsx

```diff
@@ -103,20 +104,21 @@ const perTermSections = (outputs, calculator) => {
           </a>
           )
         </div>
-
-        {terms.map(term => (
-          <CalculatorResultRow
-            key={`${section}${term.label}`}
-            id={`${section}${term.label}`}
-            label={
-              calculator.type === 'OJT' ? termLabel(term.label) : term.label
-            }
-            value={term.value}
-            bold={term.label === 'Total per year'}
-            screenReaderSpan={calculator.type === 'OJT' ? month : ''}
-            visible={term.visible}
-          />
-        ))}
+        <ul aria-labelledby={headerId}>
```

Comment:
this is missing `role="list"`

Reply:
I can't do this without suppressing a linter redundancy warning.

Reply:
If the attribute is mentioned by @1Copenut it should be added with suppression